recipes/correct-ts-specifiers/package.json

```
@@ -8,8 +8,8 @@
     "node": ">=22.15.0"
   },
   "scripts": {
-    "start": "node --no-warnings --experimental-import-meta-resolve --experimental-strip-types ./src/workflow.ts",
-    "test": "node --no-warnings --experimental-import-meta-resolve --experimental-test-module-mocks --experimental-test-snapshots --experimental-strip-types --import='../../test/snapshots.ts' --test --experimental-test-coverage --test-coverage-include='src/**/*' --test-coverage-exclude='**/*.test.ts' './**/*.test.ts'"
+    "start": "node --no-warnings --experimental-import-meta-resolve ./src/workflow.ts",
+    "test": "node --no-warnings --experimental-import-meta-resolve --experimental-test-module-mocks --experimental-test-snapshots --import='../../test/snapshots.ts' --test-coverage-include='src/**/*' --test-coverage-exclude='**/*.test.ts' --test './**/*.test.ts'"
```

```suggestion
    "test": "node --no-warnings --experimental-import-meta-resolve --experimental-test-module-mocks --experimental-test-snapshots --import='../../test/snapshots.ts' --test-coverage-exclude='**/*.test.ts' --test './**/*.test.ts'"
```

Are we importing files not in the `src/` directory? If not, this flag probably isn't needed.